e2e/tests/datasets/dataset-export-r-notebook.spec.ts

```diff
@@ -18,9 +18,7 @@ describe('Export Dataset to Notebook Test', () => {
    * Test:
    * - Export dataset to a notebook. Run the notebook code and verify run results.
    */
-  /*Skipping the test below as they will be moved to the new version of e2e test.
-   * Story tracking this effort: https://precisionmedicineinitiative.atlassian.net/browse/RW-8763*/
-  test.skip('Export to R kernel notebook in Build Dataset page', async () => {
+  test('Export to R kernel notebook in Build Dataset page', async () => {
```

Comment:
This test failed three times in a row with three different errors:
```
  ● Export Dataset to Notebook Test › Export to R kernel notebook in Build Dataset page

    TimeoutError: waiting for function failed: timeout 240000ms exceeded

      105 |       logger.error(`FAIL: WaitUntilClose failed for modal "${modalTitle}". Modal xpath: "${this.getXpath()}"`);
      106 |       logger.error(err.stack);
    > 107 |       throw new Error(err);
          |             ^
      108 |     }
      109 |   }
      110 | }

      at NewWorkspaceModal.waitUntilClose (app/modal/base-modal.ts:107:13)
      at NewWorkspaceModal.clickButton (app/container.ts:54:7)
      at WorkspaceEditPage.clickCreateFinishButton (app/page/workspace-edit-page.ts:483:5)
      at WorkspacesPage.createWorkspace (app/page/workspaces-page.ts:126:26)
      at createWorkspace (utils/test-utils.ts:163:3)
      at Object.<anonymous> (tests/datasets/dataset-export-r-notebook.spec.ts:22:5)
```

```
  ● Export Dataset to Notebook Test › Export to R kernel notebook in Build Dataset page

    waitForFunction failed: frame got detached.

      at DOMWorld._detach (node_modules/puppeteer/src/common/DOMWorld.ts:137:9)
      at Frame._detach (node_modules/puppeteer/src/common/FrameManager.ts:1286:21)
      at FrameManager._removeFramesRecursively (node_modules/puppeteer/src/common/FrameManager.ts:401:11)
      at FrameManager._onFrameDetached (node_modules/puppeteer/src/common/FrameManager.ts:350:21)
      at node_modules/puppeteer/src/common/FrameManager.ts:99:12
      at node_modules/puppeteer/vendor/mitt/src/index.ts:81:75
          at Array.map (<anonymous>)
      at Object.emit (node_modules/puppeteer/vendor/mitt/src/index.ts:81:56)
      at CDPSession.emit (node_modules/puppeteer/src/common/EventEmitter.ts:97:18)
      at CDPSession._onMessage (node_modules/puppeteer/src/common/Connection.ts:288:12)
```

```   
  ● Export Dataset to Notebook Test › Export to R kernel notebook in Build Dataset page

    TypeError: Cannot read properties of null (reading 'waitForFunction')

      63 |     await waitForDocumentTitle(this.page, this.pageTitleRegex);
      64 |     const iframe = await this.getIFrame(timeout);
    > 65 |     await iframe.waitForFunction(
         |                  ^
      66 |       (css, name) => {
      67 |         const element = document.querySelector(css);
      68 |         return element && element.textContent === name;

      at NotebookPage.isLoaded (app/page/notebook-page.ts:65:18)
      at NotebookPage.waitForLoad (app/page/authenticated-page.ts:34:5)
      at NotebookPreviewPage.openEditMode (app/page/notebook-preview-page.ts:46:5)
      at Object.<anonymous> (tests/datasets/dataset-export-r-notebook.spec.ts:55:26)
```

Reply:
These test enables accessing the notebook list, i understand the concern but since it was 3 different errors it could be flaky,. i would suggest merging this in to alteast have notebook test running and another PR to address the flaky tests